build.sh

```diff
@@ -13,7 +13,6 @@ set -e
 set -o pipefail
 
 parse "$@"
-yarn ${YARN_OPTS}
```

Comment:
why removing it ?

Reply:
To avoid building che-theia twice.

Reply:
It was decoupled to separate PR https://github.com/eclipse-che/che-theia/pull/1279